bundle/1.0.0/hive-operator.v1.0.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,620 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
Really sorry Greg I know you're rushing, can we get this into bundle/v1.0.0/, from past experience it's best we get a consistent way of writing the version otherwise we'll all be guessing and trial and erroring for all time. We should use v1.0.0 everywhere.